includes/REST_API/Hotlinking_Controller.php

```diff
@@ -294,24 +303,4 @@ public function validate_url( $value ) {
 
 		return true;
 	}
-
-	/**
-	 * Retrieves the query params for the posts collection.
-	 *
-	 * @since 1.11.0
-	 *
-	 * @return array Collection parameters.
-	 */
-	public function get_collection_params(): array {
```

Comment:
Don't understand why this was removed? I overroad this method, by design. See the original method. 

https://github.com/WordPress/wordpress-develop/blob/793bde1af74ad24c407d3645c788ac49ea352fa6/src/wp-includes/rest-api/endpoints/class-wp-rest-controller.php#L333-L367

We don't support page, per_page and search, which is why I overroad it.

Reply:
We should do this in all our customizer controller.

Reply:
`get_collection_params()` was only used in the `register_rest_route()` call further above and nowhere else. I am inlining the code instead of having this extra method. Functionality-wise nothing changes.